service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -194,16 +198,20 @@ public List<String> listValidRegions(UUID workspaceId, String platform) {
     return new ArrayList<>();
   }
 
-  public TpsLocation getLocationInfo(String platform, String location) {
+  public TpsLocation getLocationInfo(ApiCloudPlatform platform, String location) {
     features.tpsEnabledCheck();
     TpsApi tpsApi = policyApi();
     try {
-      return tpsApi.getLocationInfo(platform, location);
+      return tpsApi.getLocationInfo(convertApiCloudPlatformToTpsCloudPlatform(platform), location);
     } catch (ApiException e) {
       throw convertApiException(e);
     }
   }
 
+  private static String convertApiCloudPlatformToTpsCloudPlatform(ApiCloudPlatform platform) {
```

Comment:
No, not like this. Now you are assuming that the strings and enums will always line up.
Please do this by putting another field into the WSM `CloudPlatform` enum. We convert from API -> CloudPlatform enum -> TPS platform string.

Reply:
done